tests/testapp/tests.py

```diff
@@ -40,6 +40,16 @@ def test_validate_org_relation_error(self):
         with self.assertRaises(ValidationError):
             c.full_clean()
 
+    def test_validate_reverse_org_relation(self):
+        org1 = self._create_org(name='org1')
+        org2 = self._create_org(name='org2')
+        t = Template.objects.create(name='test-t', organization=org1)
+        Config.objects.create(name='test-c1', template=t, organization_id=str(org1.pk))
```

Comment:
use `organization=org1` instead of `organization_id=str(org1.pk)` (for consistency with the previous line, which is also easier to read).